bundles/extension.rst

```diff
@@ -129,17 +129,10 @@ read more about it, see the ":doc:`/bundles/configuration`" article.
 Adding Classes to Compile
 -------------------------
 
-.. versionadded:: 3.3
-    This technique is discouraged and the ``addClassesToCompile()`` method was
-    deprecated in Symfony 3.3 because modern PHP versions make it unnecessary.
```

Comment:
Unless I'm wrong, this versionadded says "Don't use what's descriped below." (so it's more a `.. caution::` than versionadded). This means the article should be removed (as the feature got removed in Symfony 4).

Reply:
I though the same ... but only `addClassesToCompile()` is deprecated/removed. `addAnnotatedClassesToCompile()` is still valid and used in Sf4.